.gitignore

```diff
@@ -1,5 +1,8 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
+*.swp
+package-lock.json
```

Comment:
This shouldn't be put in .gitignore, it is apparently preferable to track it, at least that is what @tacaswell found and checked it in. I am not sure how often we want/need to commit updates.